src/main/java/ch/njol/skript/conditions/CondItemLifetime.java

```diff
@@ -0,0 +1,55 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item Has Unlimited Lifetime")
+@Description("Checks if the dropped item has unlimited lifetime enabled or disabled.")
+@Examples({
+	"if all dropped items have unlimited lifetime disabled:",
+		"\tenable unlimited lifetime for all dropped items"
+})
+@Since("INSERT VERSION")
+public class CondItemLifetime extends Condition {
```

Comment:
this should use lifespan, lifetime is not a word

Reply:
> lifetime is not a word
<img width="398" alt="Screenshot 2024-12-17 at 2 22 30 PM" src="https://github.com/user-attachments/assets/a6ca7937-d473-4e68-9a55-5c60aabb1081" />

Im assuming they chose this word as that is what the Bukkit method is.

Reply:
in this context, personally, i'd prefer using lifespan, as `unlimited lifespan` is used instead of `unlimited lifetime`, but im okay with `(infinite|unlimited) (lifespan|lifetime)`

Reply:
Oh yeah, now that you say it with "unlimited"... "lifespan" makes more sense.
But yeah, having a bunch of options is good too. Wont hurt anyone.